packages/nimble-components/src/tooltip/tests/tooltip.spec.ts

```diff
@@ -123,84 +134,84 @@ describe('Tooltip', () => {
         await disconnect();
     });
 
-    // Firefox skipped, see: https://github.com/ni/nimble/issues/1075
-    it('should render the default state when selected #SkipFirefox', async () => {
+    it('should render the default state when selected', async () => {
         element.visible = true;
 
         await connect();
         await waitForUpdatesAsync();
+        await waitUntilLoaded(element);
```

Comment:
It looks like the set of tests doing `waitForUpdatesAsync()` and `waitUntilLoaded()` are the same. Could you either move `waitForUpdatesAsync()` inside `waitUntilLoaded()` or, even better, remove it entirely if it's no longer needed?

Reply:
Still needed. Moved it into the new function.